client/app/views/popover_screens/guests.coffee

```diff
@@ -70,6 +70,15 @@ module.exports = class GuestPopoverScreen extends EventPopoverScreenView
             @$('input[name="guest-name"]').focus()
 
 
+    # Link directly click action to related guest, avoid using Backbone's
+    # default event managing, as we are here in a sort of CollectionView.
+    bindGuestActions: ($element, guest) ->
+        $element
```

Comment:
Oops, sorry for my late response.

* About unbinding events : as onRemoveGuest trigger a call to `render()`, the whole guest list is rendered again, `$element` is removed and also are all event listeners bound to it. So I think that explicitly remove every event listener is not necessary.

* Using default Backbone's event managing forces us to use DOM and data-* attributes tricks to know what to do when a click event occurs, and in my opinion this is an absolute anti-pattern: 
** It relies logic behavior to DOM, and DOM is mutable an un-trustable. For example, the main concern of this PR was to simplify layout for guest list, and as I changed a few elements in the HTML structure, the buttons were not working anymore. That why I changed this part of the code, it was not my will at the beginning. Now, the only condition to have a working code is to keep the CSS classes on buttons. The rest of the HTML can change without affecting how event handlers are referencing related guests.
** It stores an index in the DOM, we have to retrieve it back in every event handler. Also, it's error prone, a part of code may have side effect on the guest array and "desynchronize" it with the index list stored in DOM. Relying every event handler with a given guest is more robust in the long term. All the logic is kept in the JS code. Also, methods are more coherent with their signatures : `onRemoveGuest` take a guest as parameter, not an index. Functionnally, we want to remove a guest, not an index.
** It decrease function purity, referencing an external component (the DOM).
* About View Collection : yes, that's was my first thought, and in my opinion the cleaner way to do that, but:
** The view is already extending `PopoverScreenView` (inheritance is a pain). I thank about just adding the useful methods in `GuestScreenView.prototype` but it started to sound a little bit over-designed and a little bit time consuming to make all the thing works, the goal was just to fix the layout, and a few `append()` may do the job.
** A another way should be to use an encapsuled `CollectionView`, but again it should imply to switch from guest as "simple object" to Backbone objects, with all the possible side effects in all the application, and I judged that the goal of the PR did not worth a huge change like this.